src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -80,29 +82,32 @@ const RegistrationForm: FunctionComponent<Props> = ({
               setFormIsFilling(false);
               setFormValues(values);
 
+              // TODO: Fix getSupportedChildData so that it works both here
+              // and in ProfileChildrenList.
               const backendSupportChildren = values.children.map(child =>
```

Comment:
revert changes in this file which not related to notification. Keep ticket scope small

Reply:
Ok. Good way to test that the toast works, when you are unable to register for the site.